db/migrate/20090915030726_change_report_field_type_to_text.rb

```diff
@@ -1,13 +1,7 @@
 class ChangeReportFieldTypeToText < ActiveRecord::Migration[4.2]
```

Comment:
We usally keep migrations empty instead of removing them for consistency. Most likely deleting would be just fine, but I'd rather not tackle that as part of this PR. There are many other empty migrations as well, might be good to squash and clean all of them in one go.